crates/egui/src/widgets/mod.rs

```diff
@@ -24,6 +24,7 @@ pub mod text_edit;
 
 #[expect(deprecated)]
 pub use self::selected_label::SelectableLabel;
+#[expect(deprecated)]
```

Comment:
this is to silence the lint right?
please add a reason
```suggestion
#[expect(deprecated, reason = "Deprecated in egui 0.33.0")]
```